wrappers/services.go

```diff
@@ -631,6 +696,9 @@ type AddSnapServicesOptions struct {
 	// the OOM killer when OOM conditions are reached.
 	VitalityRank int
 
+	// QuotaGroup is the quota group for all services in the specified snap.
+	QuotaGroup *quota.Group
```

Comment:
we should switch to carray a SnapServiceOptions inside this I suppose, it would clean some code

Reply:
I'll look at this tomorrow morning

Reply:
as discussed I will leave more TODO's about fixing up this situation, but we would likely be best served by doing the full thing and getting rid of both AddSnapServices and AddSnapServicesOptions at the same time after this week